frontend/tests/sidebar.unit.test.jsx

```diff
@@ -12,7 +12,7 @@ describe("Sidebar", () => {
 			</MemoryRouter>,
 		);
 
-		expect(screen.getByText(/pcubed/i)).toBeInTheDocument();
+		expect(screen.getByText(/point b_ank/i)).toBeInTheDocument();
```

Comment:
Welcome @DanielleDPowell ! I figured you'd be the one to ask about these tests. We'll have to meet up sometime and I'll bring you up to speed on what we've got planned - There are a couple of things that I'll for sure need your help with!